UPGRADE-3.x.md

```diff
@@ -4,6 +4,21 @@ UPGRADE 3.x
 UPGRADE FROM 3.x to 3.x
 =======================
 
+### Remove deprecated calls
+
+- `Sonata\Doctrine\Model\PageableManagerInterface` is no longer used in profit of `Sonata\DatagridBundle\Pager\PageableInterface`
+
+### Dependencies
+
+- drop support for `sonata-project/datagrid-bundle` < ^3.0.
+
+    If you are extending these method you MUST add argument and return type declarations:
```

Comment:
BTW, we always consider this kind of change as a BC break, thus this PR should target `master`.
```suggestion
    If you are extending these methods, you MUST add argument and return type declarations:
```